### PR TITLE
Add Schibsted code style

### PR DESCRIPTION
**Pull request by Sloy**
I copied our internal code style settings, based on Square and adapted for the Android's official Kotlin style guide.
Also added a little paragraph in the readme about contributing and how to install the code style.

Is this enough? Should we add or change anything else?

Fixes https://github.com/SchibstedSpain/Barista/issues/154

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -354,5 +355,12 @@ In order to speed up testing, Barista keeps in mind some considerations.
 - **Scrolls on all views**: Barista scrolls on all scrollable views, including `NestedScrollView`. Espresso only handles `ScrollView` and `HorizontalScrollView`, so people need to open questions on [StackOverflow like this](https://stackoverflow.com/questions/35272953/espresso-scrolling-not-working-when-nestedscrollview-or-recyclerview-is-in-coor). Or... just use **Barista**.
 - **Just interacts with displayed Views**: Interacting with `View`s inside a `ViewPager` throws `AmbiguousViewMatcherException`, because the views you interact with will be potentially repeated on different pages. Barista only interacts with displayed widgets, so you can focus on the behavior instead of wasting time on details.
 
+# Contributing
+
+We welcome contributions! If you found a bug or have a feature request, feel free to [open an issue](https://github.schibsted.io/scmspain/android-ij--app/issues/new) to discuss it. Remember that bugs reported with a reproducible test are more likely to be investigated and fixed. You can also submit a Pull Request.
```
Comment (danieldisu) [inline README.md]: Link is pointing to the enterprise GH

Comment (Sloy) [reply README.md]: That was dumb 😂 Fixed now, thanks!

**Comment by Sloy** [comment]
@alorma right, we already have a checkstyle config that runs on Travis, but it might be outdated. Let me check.

**Comment by rocboronat** [comment]
So good! Gogogo for me!

**Comment by rocboronat** [comment]
@Sloy what do you think about renaming the style "BaristaAndroid"? It will be simpler.

**Pull request merged**